<launch>

    <arg name="camera_config" />
    <arg name="joystick_config" />

    <!-- Launch the camera -->
    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
        <rosparam file="$(arg camera_config)" command="load" />
    </node>
    
	<!-- Launch the joy stick node -->
    <node pkg="joy" type="joy_node" name="joy_node">
         <rosparam file="$(arg joystick_config)" command="load" />
    </node>

    <!-- Launch the locobot with only use_base -->
    <include file="$(find kobuki_node)/launch/minimal.launch">
    </include> 
    

</launch>


